src/components/manage/Blocks/DataFigure/View.jsx

```diff
@@ -2,66 +2,25 @@
  * View block.
  * @module components/manage/Blocks/DataFigure/View
  */
-import './less/public.less';
+
 import Png from './Png';
 import PropTypes from 'prop-types';
-import { Transition, Button, Divider } from 'semantic-ui-react';
-import replaceSVG from '@plone/volto/icons/replace.svg';
-import { Icon } from '@plone/volto/components';
+import ViewBoX from '@eeacms/volto-image-zoom-and-flip/ImageBlock/View';
```

Comment:
Please don't add dependency to `volto-image-zoom-and-flip` as it's overriding the default volto ImageBlock and that implementation is based on an older version of volto ImageBlock.

Thus, try to extract the needed flip effect within this add-on